datahub-web-react/src/app/entity/shared/containers/profile/header/AddDeprecatedDataModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Button, DatePicker, Form, Input, message, Modal } from 'antd';
+import { useUpdateDeprecationMutation } from '../../../../../../graphql/mutations.generated';
+
+type Props = {
+    urn: string;
+    visible: boolean;
+    onClose: () => void;
+    refetch?: () => Promise<any>;
+};
+
+export const AddDeprecatedDataModal = ({ urn, visible, onClose, refetch }: Props) => {
```

Comment:
Minor: Let's change name to "DeprecationDetailsModal"

Reply:
Done and Committed.